### PR TITLE
Add Lesson model

### PR DESCRIPTION
**Pull request by WillTaylor22**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.

lesson

### CODE REVIEW

File: core/urls.py [hunk 1]
```diff
@@ -47,3 +48,9 @@ def anonymous_user_required(function):
         name='api-create-company'
     ),
 ]
+
+urlpatterns += [
+    path('lessons/', learn_views.lesson_list, name='lesson_list'),
+    path('lesson/<int:id>/', learn_views.lesson_details, name='lesson_details'),
+    path('lesson/<int:id>/delete/', learn_views.lesson_delete, name='lesson_delete'),
+]
```
Comment (sdonk) [inline core/urls.py]: Should these lines be wagtail url entries?

File: learn/admin.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+# Register your models here.
+
+from learn.models import Lesson
+admin.site.register(Lesson)
```
Comment (sdonk) [inline learn/admin.py]: Wagtail doesn't need explicit admin model registering

File: learn/models.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        Topic,
+        models.SET_NULL,
+        blank=True,
+        null=True,)
+    position = PositionField(collection='topic')
+    update_date = models.DateTimeField(auto_now=True)
+    create_date = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ['-id']
```
Comment (sdonk) [inline learn/models.py]: Should these be wagtail models?
Maybe orderable https://docs.wagtail.io/en/v2.8/reference/pages/model_reference.html#wagtail.core.models.Orderable

File: learn/views.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
+from django.shortcuts import render, get_object_or_404
+from django.middleware.csrf import get_token
+from django.http import JsonResponse
+from learn.models import Lesson
+from learn.forms import LessonForm
+
+
+def lesson_list(request, template='lesson/list.html'):
```
Comment (sdonk) [inline learn/views.py]: Should these be wagtail views? In that case you don't need the form class either

File: core/urls.py (hunk 1 above)
Comment (WillTaylor22) [reply core/urls.py]: Wagtail sounds like a good way to proceed. This was initially meant to be something quick and easy to test the backend with, but it makes sense that we'd want to use it in Wagtail.

Comment (WillTaylor22) [reply core/urls.py]: And additionally, pre-designs, to have a simple user-facing page to work with.

**Pull request merged**